Cargo.toml

```diff
@@ -4,14 +4,14 @@ version = "0.1.0"
 authors = ["XYunknown <developement@kai-greshake.de>"]
 
 [dependencies]
-chrono = "0.2"
+chrono = "^0.4"
 lazy_static = "^0.2"
 serde = "^1.0"
 serde_derive = "^1.0"
 serde_json = "^1.0"
 toml = "^0.4"
-clap = "2.23.2"
-uuid = { version = "0.4", features = ["v4"] }
+clap = "^2.25"
+uuid = { version = "^0.5", features = ["v4"] }
 dbus = "0.5.2"
```

Comment:
Have you checked the other dependencies for updates? I think we should prefix them with `^` even if they don't have updates.